package/obs/rmt-server.changes

```diff
@@ -1,3 +1,9 @@
+-------------------------------------------------------------------
+Wed Oct 30 09:01:32 UTC 2024 - Natnael Getahun <natnael.getahun@suse.com>
+
+- Version 2.20 (unreleased)
```

Comment:
I think it also includes Adnilson's schema changes.

Reply:
yes, and, at least, these PRs (for the Pubcloud)
#1225
#1226